@xen-orchestra/lite/src/components/Button.tsx

```diff
@@ -6,8 +6,8 @@ interface ParentState {}
 interface State {}
 
 interface Props {
-  label: string
-  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+  children?: React.ReactNode
+  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void
```

Comment:
Why `onClick` can be `undefined` ?
In which case we would like to use the `Button` component without his `onClick` function ?

Reply:
`onClick` could be (temporarily) `undefined` for a few reasons (e.g.: a disabled button) but I'm mostly trying to be as close as possible from a `button` which doesn't require an `onClick` or `children`.